zmon_worker_monitor/builtins/plugins/scalyr.py

```diff
@@ -124,20 +125,28 @@ def facets(self, filter, field, max_count=5, minutes=30, prio='low'):
         j = r.json()
         return j
 
-    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low'):
+    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low', align=0):
```

Comment:
@vetinari let's switch this to align=30 by default and see what happens in longer periods. I would rather abstract this part from the User until they really hit a problem with it.